src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    sortArray(this, compareFunction);
+
+    return this;
+
+    function sortArray(originalArray, method) {
+      for (let i = 0; i < originalArray.length; i++) {
+        for (let j = i + 1; j < originalArray.length; j++) {
+          if (method(originalArray[i], originalArray[j]) > 0) {
```

Comment:
The comparison function should return a negative number if the first argument is less than the second, zero if they're equal, and a positive number if the first is greater. Ensure that the compareFunction adheres to this contract for all possible inputs.